test/features/join.ts

```diff
@@ -42,43 +42,43 @@ export default function supportsJoin(
         const result = format(`
           SELECT * FROM customers
           ${join} orders ON customers.customer_id = orders.customer_id
-          ${join} items ON items.id = orders.id;
+          ${join} items ON items.col1 = orders.col1;
```

Comment:
Is it really the case that unquoted `id` can't be used as a column name in DB2 i?

When I look at the reserved word lists of [LUW-version](https://www.ibm.com/docs/en/db2/11.5?topic=sql-reserved-schema-names-reserved-words) and [Z-version](https://www.ibm.com/docs/en/db2-for-zos/11?topic=words-reserved#db2z_reservedwords__newresword) the `ID` indeed doesn't appear in reserved words list.

However I still find it surprising that a) one would need to quote such a commonly used column name b) there's such a striking difference between the i-version of DB2 from other DB2 dialects.

Reply:
The [reserved words list](https://www.ibm.com/docs/en/i/7.5?topic=words-reserved) for DB2 for IBM i shows `ID` as a reserved word - but I haven't tested if `ID` is allowed in the context of a column name.

Reply:
I stumbled into the `ID` problem when I activated `JOIN ... USING` - suddenly the tests failed for DB2i because `ID` was converted to uppercase by the formatter. So I changed the `id` in the test to a more generic column name.

Just tested `id` ad a column name on DB2 for i - and it is indeed allowed... Is it better to remove `ID` from the reserved words list?

Reply:
Could you please test this?

I remember seeing `ID` in the keywords list of some other SQL dialects, but it has always turned out that it's not really a reserved keyword.

Reply:
Yep. Best to just remove it from reserved words list. Given that this list has currently like 500 words, my suspicion is that many more of these keywords are actually not reserved.

Reply:
I encountered a similar problem when trying to find out which keywords does SingleStoreDB actually reserve. Their documentation only listed all the keywords and didn't distinguish which ones are reserved and which ones were not. I ended up executing hundreds of SQL queries to see which would actually result in a failure.